src/styles/card.scss

```diff
@@ -0,0 +1,109 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #F3F3F3;
+  background-color: #FFF;
+  border-radius: 5px;
+
+  &__img {
+    background-image: url(../images/imac.jpeg);
+    background-size: cover;
+    background-position: center;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__title {
+    font-family: Roboto, sans-serif;
```

Comment:
Move font family to the html selector

Reply:
isn't fixed. move this rule to `html` tag selector